migration.go

```diff
@@ -13,6 +14,7 @@ import (
 var (
 	now = time.Now().UTC()
```

Comment:
Let's delete this variable it's only used in a single function and is initialized every time you load the package.